gluefactory/datasets/image_folder.py

```diff
@@ -36,7 +36,7 @@ def _init(self, conf):
                     raise ValueError(
                         f"Could not find any image in folder: {conf.images}."
                     )
-                self.images = [i.relative_to(conf.images) for i in self.images]
+                self.images = [i.relative_to(conf.images) for i in self.images]  #Todo: relative paths lead to FileNotFoundError
```

Comment:
Do you have a suggestions of how we could fix this, i.e. is there any issue with just using the absolute path? That might also make the correction in __getitem__ redundant